recipes/newrelic/apm/java/linux.yml

```diff
@@ -16,85 +16,49 @@ installTargets:
     kernelArch: x86_64
     platformFamily: "debian"
 
+dependencies:
+  - infrastructure-agent-installer
 
 keywords:
   - Apm
   - java
 
 logMatch:
   - name: tomcat-versioned-catalina
-    file: /opt/tomcat/*tomcat*/logs/catalina.out
-  - name: tomcat-catalina-opt
-    file: /opt/*tomcat*/logs/catalina.out
-  - name: tomcat-catalina-var
-    file: /var/log/*tomcat*/catalina.out
+    file: /opt/tomcat/*tomcat*/logs/*.log
```

Comment:
@mbruzina why are we removing the other logMatch?

Reply:
That was a mistake, I'll bring it back 👍